Userland/Libraries/LibWeb/DOM/ElementFactory.cpp

```diff
@@ -431,6 +432,8 @@ static WebIDL::ExceptionOr<JS::GCPtr<SVG::SVGElement>> create_svg_element(JS::Re
         return MUST_OR_THROW_OOM(realm.heap().allocate<SVG::SVGForeignObjectElement>(realm, document, move(qualified_name)));
     if (local_name == SVG::TagNames::line)
         return MUST_OR_THROW_OOM(realm.heap().allocate<SVG::SVGLineElement>(realm, document, move(qualified_name)));
+    if (local_name == SVG::TagNames::stop)
+        return MUST_OR_THROW_OOM(realm.heap().allocate<SVG::SVGStopElement>(realm, document, move(qualified_name)));
     if (local_name == SVG::TagNames::path)
```

Comment:
These ifs are also (mostly) alphabetized.

(Not sure why some do `==` and some do `equals_ignoring_case`)

Reply:
I think SVG tag names are case sensitive, while HTML for HTML it does not matter

Reply:
Not sure about the ones like `clipPath` though :thinking: